apps/sms/views/conversation/js/conversation.js

```diff
@@ -80,6 +80,11 @@ var ConversationView = {
   showErrorInFailedEvent: '',
   previousSegment: 0,
 
+  /**
+   * Set to true after init is run.
+   */
+  _isReady: false,
```

Comment:
quite a hack, but useful to know whether init is run yet -- otherwhise at startup there is a moment where the object exists because the file was properly parsed and executed but init is not called yet.